CODEOWNERS

```
@@ -20,6 +20,6 @@ scripts/ @vicentebolea
 source/adios2/toolkit/sst/dp/mpi_dp.c @vicentebolea
 
 # GPU-aware specific files
-source/adios2/helper/adiosKokkos.* @anagainaru
+source/adios2/helper/kokkos/adiosKokkos.* @anagainaru
```

If we make a separate folder I might as well cover all files that will be here. The query logic might require some Kokkos components as well.
```suggestion
source/adios2/helper/kokkos/* @anagainaru
```